dmk2cw.c

```diff
@@ -383,9 +387,11 @@ main(int argc, char** argv)
   int dam_min, dam_max, got_iam, skip;
   int byte, bit;
   double mult;
-  int rx02_data = 0;
+  int rx02_data;
+  int sector_data;
   int cw_mk = 1;
   int tracklen;
+  int extra;
```

Comment:
The variable `extra` could be named more descriptively.  Extra what?  Maybe `extra_bytes` or `quirk_bytes`?